install/kubernetes/cilium/values.yaml

```diff
@@ -587,6 +587,10 @@ ingressController:
     insecureNodePort: ~
     # -- Configure a specific nodePort for secure HTTPS traffic on the shared LB service
     secureNodePort : ~
+    # -- Configure a specific loadBalancerClass on the shared LB service
+    loadBalancerClass: ~
```

Comment:
note: you need to edit `values.yaml.tmpl` first then run `make -C install/kubernetes` as well as `make -C Documentation update-helm-values`

Reply:
My bad, I've done this now.